### DISCUSSION TITLE
website says License is for Edge but extension build is for Chrome.  I paid separately for both.  VERY frustrating!  Won't give unlimited downloads in Edge, as it seems to think it is chrome!

### DISCUSSION THREAD

**Discussion by tuminj** [aclap-dev/video-downloadhelper#1455] [License & Purchase requests]
### What's your issue?

website says License is for Edge but extension build is for Chrome.  I paid separately for both.  VERY frustrating!  Won't give unlimited downloads in Edge, as it seems to think it is chrome!

### The 10 first digits (only 10) of your license number or email address

b732ea7dc7

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi, you have 2 valid licenses, one for Edge and one for Chrome, please make sure you use the good one on each browser.
If your Edge license says "extension build is for Chrome", you might have installed the Chrome extension on Edge. I would recommend to remove extension on Edge browser and reinstall from here: https://microsoftedge.microsoft.com/addons/detail/video-downloadhelper/jmkaglaafmhbcpleggkmaliipiilhldn

**Comment by tuminj** [comment]
Hello: when I am in chrome, it says I need  "Premium status required".  can you send me a link to re-install downloader for Chrome?  And, to remove the extension, do I need to do anything other than 'remove"?  thanks.

**Reply by jcfrog** [reply]
Just remove it.
All the links are here: https://github.com/aclap-dev/video-downloadhelper/wiki#1-the-extension

**Comment by PoppaTango** [comment]
It's doing the same thing for me as well.  

Here's the first 10 chars of my License key!

09f85b2103

**Reply by PoppaTango** [reply]
![image](https://github.com/user-attachments/assets/c673a71a-e704-4f4c-a67a-e8e78ca27b65)

**Reply by jcfrog** [reply]
You may have installed the wrong extension. 
I recommend that you remove the extension and reinstall it from here: https://microsoftedge.microsoft.com/addons/detail/video-downloadhelper/jmkaglaafmhbcpleggkmaliipiilhldn

**Comment by sodrediana** [comment]
Ola  comprei a licença  mais ela nao permite  usar no EDGE, gostariia de saber como fazer

**Reply by jcfrog** [reply]
Hi,
please send us more details at licenses@downloadhelper.net
